src/index/ivf/ivf.cc

```diff
@@ -701,6 +749,12 @@ KNOWHERE_REGISTER_GLOBAL(IVFFLAT,
                          [](const Object& object) { return Index<IvfIndexNode<faiss::IndexIVFFlat>>::Create(object); });
 KNOWHERE_REGISTER_GLOBAL(IVF_FLAT,
                          [](const Object& object) { return Index<IvfIndexNode<faiss::IndexIVFFlat>>::Create(object); });
+KNOWHERE_REGISTER_GLOBAL(IVFFLATCC, [](const Object& object) {
```

Comment:
remove this name, keep "IVF_FLAT_CC" only

Reply:
This name will keep for python side.